tests/kernel/wave/wave_e2e_test.py

```diff
@@ -543,11 +551,14 @@ def repeat(acc: tkl.Register[M, NF, tkl.f32]) -> tkl.Register[M, NF, tkl.f32]:
 
 
 @require_e2e
-def test_igemm_conv():
-    n, c, h, w = 1, 4, 5, 5  # Image.
-    nf, cf, hf, wf = 16, c, 2, 2  # Filters.
+@pytest.mark.parametrize("n", [1, 2, 4])
+@pytest.mark.parametrize("c", [1, 3, 4, 10])
+@pytest.mark.parametrize("nf", [1, 2, 16])
+@pytest.mark.parametrize("stride", [1, 2, 3])
+def test_igemm_conv(n, c, nf, stride):
+    h, w = 5, 5  # Image.
+    cf, hf, wf = c, 2, 2  # Filters.

```

Comment:
Not for this patch, but would it work if you parameterized h, w, hf and wf?

Reply:
We should, but I need to find a better way of deciding which combination of parameters we are going to test, as currently amount of test permutations grow multiplicative.

Reply:
Makes sense. Just wanted to know if it was working and sounds like it is which is great. We can parameterize it for the nightly test.